CMakeLists.txt

```diff
@@ -197,6 +198,7 @@ endmacro(add_dynamic_ros_node)
 add_subdirectory(src/examples)
 add_subdirectory(src/utility)
 add_subdirectory(src/movement)
+add_subdirectory(src/trilateration)
```

Comment:
As the hydrophones are sensors, shouldn't this go under the `sensors` subdirectory? Especially since it is only a single .cpp file and the CMakesList file.